airbyte-integrations/connectors/destination-snowflake/src/main/kotlin/io/airbyte/integrations/destination/snowflake/operation/SnowflakeStagingClient.kt

```diff
@@ -63,8 +79,21 @@ class SnowflakeStagingClient(private val database: JdbcDatabase) {
         recordsData: SerializableBuffer
     ) {
         val query = getPutQuery(stageName, stagingPath, recordsData.file!!.absolutePath)
-        log.info { "Executing query: $query" }
-        database.execute(query)
+        val queryId = UUID.randomUUID()
+        log.info { "query $queryId, executing $query" }
```

Comment:
nit:
```suggestion
        log.info { "executing query ($queryId): $query" }
```

(slightly easier to grep for `executing query` instead of just `executing` :P )

Reply:
true but kept it consistent to grep query <uuid> to figure start and end of query or any warns